### PR TITLE
[CHORE] Update Ember Version and remove shims

### PR DESCRIPTION
**Pull request by LiquidSean**
Ember version was at 2.13. I have updated this to 2.17 and have removed shims as that is not used and is deprecated.

### CODE REVIEW

**Comment by tylerbrazier** [comment]
looks like ember 3.0 is out. do you think we may as well just get the latest?

**Comment by tylerbrazier** [comment]
Also, are we sure we need that `.idea/workspace` file?

**Comment by LiquidSean** [comment]
@tylerbrazier I have updated the Ember Version

File: .gitignore [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 # dependencies
 /node_modules
+/bower_components
```
Comment (tylerbrazier) [inline .gitignore]: does this project use bower?

**Comment by tylerbrazier** [comment]
do we need travis.yml? we use circle. also, do we need .idea/workspace.xml?

File: package.json [hunk 2]
```diff
@@ -1,66 +1,62 @@
 {
   "name": "ember-baremetrics-calendar",
-  "version": "0.0.4",
+  "version": "0.0.6",
```
Comment (tylerbrazier) [inline package.json]: looks like we jumped from 0.0.4 to 0.0.6; 0.0.5 should be good for this PR

**Comment by tylerbrazier** [comment]
LGTM 👍

**Pull request merged**